app/services/users.js

```diff
@@ -0,0 +1,8 @@
+const errors = require('../errors');
+const logger = require('../logger');
+const { User } = require('../models/user');
+
+exports.createUser = params => {
+  logger.info(`Create User Params: ${params}`);
+  return User.create(params).catch(error => Promise.reject(errors.serverError(error)));
```

Comment:
Remember to sanitize params before create, we use mappers for that, also this seems like a databaseError 😄